examples/minimal/src/main.rs

```diff
@@ -0,0 +1,99 @@
+#![recursion_limit = "256"]
+use yew::prelude::*;
+
+use yew_router::Switch;
+use yew_router::service::RouteService;
+use yew_router::route::Route;
+
+
+#[global_allocator]
+static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
+
+fn main() {
+    yew::initialize();
+    web_logger::init();
+    App::<Model>::new().mount_to_body();
+    yew::run_loop();
+}
+
+pub struct Model {
+    route_service: RouteService<()>,
+    route: Route<()>,
+}
```

Comment:
@Boscop, I've created an example that only uses the service + `Switch` derive to accomplish routing. It shows what using the library without any of the features enabled is like.